app/assets/scripts/views/location/temporal-measurements.js

```diff
@@ -0,0 +1,213 @@
+  CardTitle,
+} from '../../components/card';
+import TabbedSelector from '../../components/tabbed-selector';
+import CardList from '../../components/card-list';
+
+const ErrorMessage = styled.div`
+  grid-column: 1 / -1;
+`;
+const CardHeader = styled(BaseHeader)`
+  display: grid;
+  grid-template-rows: min-content 1fr 1fr;
+  grid-gap: 0.5rem;
+`;
+
+export default function TemporalMeasurements({ measurements, parameters }) {
+  const { fetched, fetching, error } = measurements;
+  const [activeTab, setActiveTab] = useState(parameters[0]);
+  const [activeTabMeasurments, setActiveTabMeasurments] = useState([]);
+  const [hourCoverage, setHourCoverage] = useState([]);
+  const [dayCoverage, setDayCoverage] = useState([]);
+  const [monthCoverage, setMonthCoverage] = useState([]);
+
+  useEffect(() => {
+    if (fetched) {
+      const tabMeasurements = measurements.data.results.filter(
+        f => f.parameter === activeTab.id
+      );
+      setActiveTabMeasurments(tabMeasurements);
+      setHourCoverage(parseHour(tabMeasurements));
+      setDayCoverage(parseDay(tabMeasurements));
+      setMonthCoverage(parseMonth(tabMeasurements));
+    }
+  }, [fetched, activeTab]);
+
+  if (!fetched && !fetching) {
+    return null;
+  }
+
+  const parseHour = measurements => {
+    const hourCount = measurements.reduce((prev, curr) => {
+      const hour = new Date(curr.date.local).getHours();
+      prev[hour] = prev[hour] ? prev[hour] + 1 : 1;
+      return prev;
+    }, {});
+    const orderedHoursCount = {};
+    Object.keys(hourCount)
+      .sort()
+      .forEach(function (key) {
+        orderedHoursCount[key] = hourCount[key];
+      });
+    return orderedHoursCount;
+  };
+
+  const parseDay = measurements => {
+    const daysOfWeek = ['SUN', 'MON', 'TUES', 'WED', 'THURS', 'FRI', 'SAT'];
+    const dayCount = measurements.reduce((prev, curr) => {
+      const day = daysOfWeek[new Date(curr.date.local).getDay()];
+      prev[day] = prev[day] ? prev[day] + 1 : 1;
+      return prev;
+    }, {});
+    let sortedDayCount = {};
+    daysOfWeek.forEach(day =>
+      dayCount[day]
+        ? (sortedDayCount[day] = dayCount[day])
+        : (sortedDayCount[day] = 0)
+    );
+    return sortedDayCount;
+  };
+
+  const parseMonth = measurements => {
+    const monthOfYear = [
+      'JAN',
+      'FEB',
+      'MAR',
+      'APR',
+      'MAY',
+      'JUNE',
+      'JUL',
+      'AUG',
+      'SEP',
+      'OCT',
+      'NOV',
+      'DEC',
+    ];
+    const monthCount = measurements.reduce((prev, curr) => {
+      const month = monthOfYear[new Date(curr.date.local).getMonth()];
+      prev[month] = prev[month] ? prev[month] + 1 : 1;
+      return prev;
+    }, {});
+    let sortedMonthCount = {};
+    monthOfYear.forEach(month =>
+      monthCount[month]
+        ? (sortedMonthCount[month] = monthCount[month])
+        : (sortedMonthCount[month] = 0)
+    );
+    return sortedMonthCount;
+  };
+
+  if (fetching) {
+    return <LoadingMessage />;
```

Comment:
Id recommend wrapping this in a styled comp, see measurements-card.js for example. otherwise the loading comp is not the same size as the card